include/seqan3/argument_parser/detail/format_parse.hpp

```diff
@@ -451,7 +451,7 @@ class format_parse : public format_base
         if (res == option_parse_result::error)
         {
             throw user_input_error{msg + "Argument " + input_value + " could not be parsed as type " +
-                                   get_type_name_as_string(input_value) + "."};
+                                   get_type_name_as_string(option_type{}) + "."};
```

Comment:
I guess it is fine to assume default-constructible here.

Reply:
I thought the same, but then I saw
https://github.com/seqan/seqan3/blob/055a3bea684238c29ee27977b103103a0211797f/include/seqan3/argument_parser/detail/format_parse.hpp#L380
So i guess it's fine